clients/fides-js/src/components/tcf/TcfVendors.tsx

```diff
@@ -158,13 +143,13 @@ const DataCategories = ({
   );
 };
 
-const StorageDisclosure = ({ vendor }: { vendor: Vendor }) => {
+const StorageDisclosure = ({ vendor }: { vendor: VendorRecord }) => {
   const {
     name,
-    usesCookies,
-    usesNonCookieAccess,
-    cookieMaxAgeSeconds,
-    cookieRefresh,
+    uses_cookies: usesCookies,
+    uses_non_cookie_access: usesNonCookieAccess,
+    cookie_max_age_seconds: cookieMaxAgeSeconds,
```

Comment:
It's not clear to me in the admin ui that this cookie age is seconds - 
<img width="793" alt="Screenshot 2023-10-17 at 12 27 41 PM" src="https://github.com/ethyca/fides/assets/9755598/38dfe8a0-b27b-47ac-a0fd-4081e237901c">

Reply:
yeah I brought this up here but I don't feel like we really reached a conclusion: https://ethyca.slack.com/archives/C05E74TR0BB/p1697483221897889

Reply:
I got the okay to put `Maximum duration (seconds)` here—I'll do that, but also there's another ticket to make this into a better input one day: https://github.com/ethyca/fides/issues/4293